text/069-coq-roadmap.md

```diff
@@ -0,0 +1,316 @@
+- Title: Coq roadmap
+
+- Drivers: Nicolas Tabareau (@tabareau), Théo Zimmermann (@Zimmi48)
+
+----
+
+# Summary
+
+This CEP aims to establish a roadmap for the Coq project.
+It highlights both:
+
+- what are considered as important axes to work on in the future,
+- what resources are available to work on these axes, and which axes we commit
+  to work on based on these resources.
+
+Resources mean availability of persons to conduct the work. For technical axes,
+requiring changes in the Coq system, it includes availability of persons to
+review and merge the changes.
+
+# Motivation
+
+Producing a roadmap for the Coq project is important for several reasons:
+
+- It helps the community to know what to expect from the Coq project in the
+  future, in particular what changes can reasonably be expected to become
+  part of next releases, and what changes are more likely to be delayed
+  because of lack of resources.
+
+- It helps Coq developers to focus on important axes, and to be more
+  efficient, by making sure that their work will be supported by other
+  developers, including reviewers.
+
+- It helps contributors to know what are the important axes where their
+  contribution is most likely to be welcome and where they can expect
+  the most support from the Coq developers.
```

Comment:
```suggestion
- It helps contributors to know in what areas their
  contribution is likely to get
  the most support from the Coq developers.
```

@Zimmi48 It seems unnecessary to use "welcome" at all.  Also, IMHO "axes" is a strange word choice.  It suggests that items are independent (orthogonal?), but that is probably not always the case.